GVFS/GVFS.Common/LegacyPlaceholderListDatabase.cs

```diff
@@ -273,6 +273,21 @@ public void WriteAllEntriesAndFlush(IEnumerable<IPlaceholderData> updatedPlaceho
             }
         }
 
+        public void RemoveStartingWith(string path)

```

Comment:
Remove this method.  I don't think it is used anymore.

Reply:
Looks like this method is still being used here:

```
        public bool DehydrateFolder(string relativePath)
        {
            List<IPlaceholderData> removedPlaceholders;
            List<string> removedModifiedPaths;
            bool successful = false;

            try
            {
                removedPlaceholders = this.placeholderDatabase.RemoveStartingWith(relativePath);
```

Reply:
That is using the interface version below that returns the list.